lib/kitchen/provisioner/chef_base.rb

```diff
@@ -44,6 +44,13 @@ module Provisioner
     class ChefBase < Base
       default_config :require_chef_omnibus, true
       default_config :chef_omnibus_url, "https://omnitruck.chef.io/install.sh"
+      default_config :chef_omnibus_root do |provisioner|
+        if provisioner.powershell_shell?
+          "/opt/chef" # I know that we don't need this, but it leaves things flexible.
```

Comment:
Any reason not to set this to the Windows default of `"$env:systemdrive\\opscode\\chef"`?

Reply:
I based that on content that I had seen inside of the spec tests.

spec/kitchen/provisioner/chef_base_spec.rb#L604 which looks to expect the same path as unix-like/linux systems.  I did contemplate using `provisioner.windows_os?` to `"$env:systemdrive\\opscode\\chef"` entry.

Reply:
that should be c:\opscode\chef though not /opt/chef for windows